Client/Frontend/Popup/PaymentHandlerPopupView.swift

```diff
@@ -0,0 +1,284 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import UIKit
+import BraveShared
+import Shared
+import Lottie
+
+class PaymentHandlerPopupView: PopupView {
+    fileprivate var dialogImage: UIView?
+    fileprivate var titleLabel: UILabel!
+    fileprivate var messageLabel: UILabel!
+    fileprivate var containerView: UIView!
+    fileprivate var textField: UITextField?
+    fileprivate var displayItemTitle: UILabel!
+    fileprivate var displayItems = [UILabel]()
+    fileprivate var totalItemTitle: UILabel!
+    fileprivate var totalItem: UILabel!
+    
+    var text: String? {
+        return textField?.text
+    }
+    
+    fileprivate let kAlertPopupScreenFraction: CGFloat = 0.8
+    fileprivate let kPadding: CGFloat = 20.0
+    
+    init(imageView: UIView?, title: String, message: String, inputType: UIKeyboardType? = nil,
+         secureInput: Bool = false, inputPlaceholder: String? = nil, titleWeight: UIFont.Weight = UIFont.Weight.bold, titleSize: CGFloat = 24) {
+        super.init(frame: CGRect.zero)
+        
+        overlayDismisses = false
+        defaultShowType = .normal
+        defaultDismissType = .noAnimation
+        presentsOverWindow = true
+        
+        containerView = UIView()
+        containerView.autoresizingMask = [.flexibleWidth]
+        
+        if let imageView = imageView {
+            containerView.addSubview(imageView)
+            dialogImage = imageView
+        }
+        
+        titleLabel = UILabel()
+        titleLabel.textColor = BraveUX.greyJ
+        titleLabel.textAlignment = .center
+        titleLabel.font = UIFont.systemFont(ofSize: titleSize, weight: titleWeight)
```

Comment:
Can remove the `UIFont` and just do `.systemFont`
All of these can be configured in `.then`